spanner/api/Spanner.Samples/InsertUsingDmlCoreAsync.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_dml_standard_insert]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class InsertUsingDmlCoreAsyncSample
+{
+    public async Task<int> InsertUsingDmlCoreAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        using var cmd = connection.CreateDmlCommand("INSERT Singers (SingerId, FirstName, LastName) VALUES (12, 'Virginia', 'Watson')");
```

Comment:
In the original sample [the value is 10](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/7192e70f2f6e1ee8b8b5203b8768c89913d368db/spanner/api/Spanner/Program.cs#L1363) but if there's a good reason for it being 12 here, I don't mind.